immutable/LICENSE

```diff
@@ -0,0 +1,30 @@
+BSD License
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE
this repository is MIT license.